src/category_theory/category/PartialFun.lean

```diff
@@ -0,0 +1,147 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import category_theory.category.Pointed
+import data.pfun
+
+/-!
+# The category of types with partial functions
+
+This defines `PartialFun`, the category of types equipped with partial functions.
+
+This category is classically equivalent to the category of pointed types. The reason it doesn't hold
+classically stems from the difference between `part` and `option`. Both can model partial functions,
```

Comment:
Looks like there's a typo in here, the latter should be 'constructively' not 'classically'?